### PR TITLE
Tutorial adding command

### LINKED ISSUE #55: [User Guide] Add Affiliation function

### LINKED ISSUE #65: Add Affn command
... to search for affiliations

### PR DESCRIPTION
**Pull request by EricXiong420**

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/AffiliationCommandParser.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.AffiliationCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.person.Name;
+
+/**
+ * Parses input arguments and creates a new DeleteCommand object
```
Comment (leezhanpeng) [inline src/main/java/seedu/address/logic/parser/AffiliationCommandParser.java]: May need to modify the comment.

File: src/main/java/seedu/address/logic/parser/AffiliationCommandParser.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ */
+public class AffiliationCommandParser implements Parser<AffiliationCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the DeleteCommand
+     * and returns a DeleteCommand object for execution.
```
Comment (leezhanpeng) [inline src/main/java/seedu/address/logic/parser/AffiliationCommandParser.java]: Likewise for this!

File: src/test/data/JsonSerializableAddressBookTest/typicalPersonsAddressBook.json [hunk 3]
```diff
@@ -5,25 +5,25 @@
...
     "address" : "123, Jurong West Ave 6, #08-111",
-    "tags" : [ "friends" ]
+    "affiliations" : [ "Benson Meier" ]
   }, {
     "name" : "Benson Meier",
     "phone" : "98765432",
     "email" : "johnd@example.com",
     "address" : "311, Clementi Ave 2, #02-25",
-    "tags" : [ "owesMoney", "friends" ]
+    "affiliations" : [ "Alice Pauline", "Carl Kurz" ]
```
Comment (leezhanpeng) [inline src/test/data/JsonSerializableAddressBookTest/typicalPersonsAddressBook.json:14]: So technically our affiliations can store more than one entity, but its just that during the addition of affiliation, it registers the whole string as a single person?

File: src/test/data/JsonSerializableAddressBookTest/typicalPersonsAddressBook.json [hunk 4]
```diff
@@ -5,25 +5,25 @@
...
     "address" : "wall street",
-    "tags" : [ ]
+    "affiliations" : [ ]
   }, {
     "name" : "Daniel Meier",
     "phone" : "87652533",
     "email" : "cornelia@example.com",
     "address" : "10th street",
-    "tags" : [ "friends" ]
+    "tags" : [ ]
```
Comment (leezhanpeng) [inline src/test/data/JsonSerializableAddressBookTest/typicalPersonsAddressBook.json]: Should this be "affiliations"? Likewise for the few of them further below this.

**Review by leezhanpeng — COMMENTED** [review]
LGTM! Just that some of the codes haven't pass checkstyle yet, so cannot merge due to the checks. Maybe we can discuss whether we should remove the checkstyle as part of our PR checks.

File: src/test/data/JsonSerializableAddressBookTest/typicalPersonsAddressBook.json (hunk 3 above)
Comment (EricXiong420) [reply src/test/data/JsonSerializableAddressBookTest/typicalPersonsAddressBook.json:14]: yep! exactly

File: src/test/data/JsonSerializableAddressBookTest/typicalPersonsAddressBook.json (hunk 4 above)
Comment (EricXiong420) [reply src/test/data/JsonSerializableAddressBookTest/typicalPersonsAddressBook.json]: yep! fixed this in the next commit

File: src/main/java/seedu/address/logic/parser/AffiliationCommandParser.java (hunk 2 above)
Comment (EricXiong420) [reply src/main/java/seedu/address/logic/parser/AffiliationCommandParser.java]: thanks for the keen eyes haha fixed it in the next commit

File: docs/UserGuide.md [hunk 5]
```diff
@@ -79,15 +79,16 @@ Format: `help`
 
 Adds a contact to the contact book.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL r/ROLE [a/AFFN]`
+Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [af/AFFILIATION]…​`
```
Comment (Seonlo99) [inline docs/UserGuide.md:82]: LGTM. Just that the address could be replaced with role. Same goes for the rest of the docs.

**Review by Seonlo99 — APPROVED** [review]

**Pull request merged**